migrations/sql/V10__Change_file_last_modified_column_date_type.sql

```diff
@@ -0,0 +1,7 @@
+/**
+  Switch the file table from using string last_modified_dates to timestamp with timezone to store dates correctly.
+ */
+
+ALTER TABLE file
+  ALTER COLUMN last_modified_date
+  TYPE TIMESTAMPTZ USING last_modified_date::TIMESTAMPTZ;
```

Comment:
Will the migration succeed if there's already some data in this column? It doesn't really matter how the dates are converted (since the DB just has dummy data the moment) as long as they're valid after the migration.

Reply:
Added additiional steps to create new column, add "dummy" timestamps and replace existing last_modified_date with new column.